WeakAurasOptions/ConditionOptions.lua

```diff
@@ -1548,48 +1548,33 @@ local function addControlsForIfLine(args, order, data, conditionVariable, totalA
   end
 
   if (currentConditionTemplate and currentConditionTemplate.type and type(currentConditionTemplate.type) == "string") then
-    local setOp;
-    local setValue;
-    if (data.controlledChildren) then
-      setOp = function(info, v)
-        check = getOrCreateSubCheck(conditions[i].check, path);
-        for id, reference in pairs(conditions[i].check.references) do
-          local auraData = WeakAuras.GetData(id);
-          local childCheck = getOrCreateSubCheck(auraData[conditionVariable][reference.conditionIndex].check, path);
-          childCheck.op = v;
-          WeakAuras.Add(auraData);
-          OptionsPrivate.ClearOptions(auraData.id)
+    local function makeSetter(field)
```

Comment:
I renamed this from "makeSet" to "makeSetter", as that is the proper name for a set function.